addon/components/sl-calendar.js

```diff
@@ -161,18 +151,18 @@ export default Ember.Component.extend({
         /**
          * Set the current year
          *
-         * @function actions.setYear
-         * @param    {number} year - The year to set to the current value
-         * @returns  {void}
+         * @function actions:setYear
+         * @param {Number} year - The year to set to the current value
```

Comment:
Do we want a "-" separating the param name and the description? Think this will cause the description to then display as "- ..." in the generated documentation.